modules/apps/frontend-js/frontend-js-alert-support-web/src/main/java/com/liferay/frontend/js/alert/support/web/internal/servlet/taglib/AlertBottomDynamicInclude.java

```diff
@@ -38,16 +37,8 @@ public void include(
 			HttpServletResponse httpServletResponse, String key)
 		throws IOException {
 
-		ScriptData scriptData = new ScriptData();
-
-		String initModuleName = _npmResolver.resolveModuleName(
-			"frontend-js-alert-support-web/index");
-
-		scriptData.append(
-			null, "AlertProvider.default()",
-			initModuleName + " as AlertProvider", ScriptData.ModulesType.ES6);
-
-		scriptData.writeTo(httpServletResponse.getWriter());
+		_jsModuleLauncher.writeModuleInvocation(
+			httpServletResponse.getWriter(), "frontend-js-alert-support-web");
```

Comment:
Not for this iteration, but if we're also going to be invoking own modules, we could end up doing something similar to that of `npmResolver` so we don't even need to pass in the module name.

Reply:
Even though we did pass the module name to `npmResolver`, but doesn't make sense, should be able to get it from somewhere within here so that we could avoid having to hardcode it. This is still one of those strings that is often hard to understand where it's coming from.

Reply:
Yep. Most probably. :+1: